manifests/init.pp

```diff
@@ -65,6 +65,7 @@
   Stdlib::IP::Address::Nosubnet $api_service_address = k8s::ip_in_cidr($service_cluster_cidr, 'first'),
   K8s::IP_addresses $dns_service_address             = k8s::ip_in_cidr($service_cluster_cidr, 'second'),
   Stdlib::Fqdn $cluster_domain                       = 'cluster.local',
+  String[1] $etcd_cluster_name                       = 'default',
```

Comment:
Not sure if this needs to be on the main `k8s` class, as it's only ever used by the server components.

Reply:
in most other modules one can set all data via the main class. so if i want to use k8s i know i can set everything over k8s::something and have not to look in any other class if there might be a parameter i have to set to change something. but this is only design philosophy.